src/commands/use.js

```diff
@@ -58,39 +58,49 @@ module.exports = new Command("use [alias_or_project_id]")
     if (newActive) {
       // firebase use [alias_or_project]
       var aliasedProject = options.rc.get(["projects", newActive]);
-      return api.getProjects().then(function(projects) {
-        if (aliasOpt) {
-          // firebase use [project] --alias [alias]
-          if (!projects[newActive]) {
-            return utils.reject(
-              "Cannot create alias " + clc.bold(aliasOpt) + ", " + verifyMessage(newActive)
-            );
+      var project = null;
+      const lookupProject = aliasedProject || newActive;
+      return firebaseApi
+        .getProject(lookupProject)
+        .then(function(foundProject) {
```

Comment:
Let's go ahead and be consistent with arrow functions since we're doing it elsewhere -- not your fault, we're gradually transitioning this codebase to modern JS and it gets a little weird when old files get touched for the first time in forever 😄

Reply:
:+1: - Gotta love the ever evolving cat and mouse game that is JavaScript development.
Fixing this now.